Project.toml

```diff
@@ -10,11 +10,11 @@ QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-julia = "1"
+julia = "1.6"
```

Comment:
Note that such change requires bumping the minor version number of the package (i.e. 1.6.0)

Reply:
Sure; I just thought changing myself your version number would be presumptuous.